src/LinkBubbleMenu/EditLinkMenuContent.tsx

```diff
@@ -178,6 +184,11 @@ export default function EditLinkMenuContent({
             formatHref();
           }
         }}
+        slotProps={{
+          htmlInput: {
+            pattern: "(http|https|mailto|tel):\\/\\/.*|^/.*",
```

Comment:
Per https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern#overview, it seems the `^` is redundant/unneeded:

> The pattern's regular expression must match the entire input's value, rather than matching a substring - as if a ^(?: were implied at the start of the pattern and )$ at the end.

Confirmed in testing locally that it still works to require a `/` at the start for relative URLs, even without `^`.